docs/timezone.md

```diff
@@ -133,12 +135,77 @@ tz2 = Temporal.TimeZone.from(tz);
 ### timeZone.**id** : string
 
 The `id` property gives an unambiguous identifier for the time zone.
-Effectively, this is the canonicalized version of whatever `timeZoneIdentifier` was passed as a parameter to the constructor.
+This is the normalized version of whatever `timeZoneIdentifier` was passed as a parameter to the constructor.
 
 When subclassing `Temporal.TimeZone`, this property must be overridden to provide an identifier for the custom time zone.
 
 ## Methods
 
+### timeZone.**equals**(_other_: Temporal.TimeZone | object | string) : boolean
+
+**Parameters:**
+
+- `other` (`Temporal.TimeZone` object, object implementing the `Temporal.TimeZone` protocol, or a string time zone identifier): Another time zone to compare.
+
+**Returns:** `true` if `timeZone` and `other` are equal, or `false` if not.
+
+Compares two time zones for equality.
+Equality is determined by the following algorithm:
+
+- If `timeZone === other`, then the time zones are equal.
+- Otherwise, `timeZone.id` is compared to `other` (or `other.id` if `other` is an object).
+  If any of the following conditions are true, then the time zones are equal:
+  - If both string identifiers are Zone or Link names in the [IANA Time Zone Database](https://www.iana.org/time-zones), and they resolve to the same Zone name.
+    This resolution is case-insensitive.
+  - If both string identifiers are custom time zone identifiers and both identifiers are equal.
+    This is a case-sensitive comparison.
```

Comment:
```suggestion
    This comparison is equivalent to `===`, and therefore sensitive to both case and character normalization.
```

Reply:
Happy to adjust this text, but I think your suggestion is a bit too technical for the layperson who may not know what "character normalization" is, and who may not recognize "sensitive to case" instead of the more familiar "case-sensitive".  What about this?

>  - If both string identifiers are custom time zone identifiers that are equal according to `===`.
    This comparison is case-sensitive and does not normalize Unicode characters.

Reply:
Or this, even simpler language: 

>   - Both string identifiers are custom time zone identifiers that are equal according to `===`.
    This comparison is case-sensitive and treats every Unicode character as different.

Reply:
Hopefully this is OK because I included this text in the latest push. :-)

Reply:
"_comparison is case-sensitive and does not normalize Unicode characters_" is reasonably good, but "_comparison is case-sensitive and treats every Unicode character as different_" is not IMO. Relevant examples are things like \<U+00E9
LATIN SMALL LETTER E WITH ACUTE> vs. \<U+0065 LATIN SMALL LETTER E, U+0301 COMBINING ACUTE ACCENT> (`"é" !== "é"`) where decomposition matters, or \<U+0061 LATIN SMALL LETTER A, U+0301 COMBINING ACUTE ACCENT, U+0323 COMBINING DOT BELOW> vs. \<U+0061 LATIN SMALL LETTER A, U+0323 COMBINING DOT BELOW, U+0301 COMBINING ACUTE ACCENT> (`"ạ́" !== "ạ́"`) where the ordering of combining characters matters.

Reply:
> "_comparison is case-sensitive and does not normalize Unicode characters_" is reasonably good

OK, I'll use that.